parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -165,14 +165,19 @@ configureK8s() {
     chmod 0644 "${APISERVER_PUBLIC_KEY_PATH}"
     chown root:root "${APISERVER_PUBLIC_KEY_PATH}"
 
-    AZURE_JSON_PATH="/etc/kubernetes/azure.json"
-    touch "${AZURE_JSON_PATH}"
-    chmod 0600 "${AZURE_JSON_PATH}"
-    chown root:root "${AZURE_JSON_PATH}"
-
-    set +x
```

Comment:
I think we still need this `set +x` here to avoid printing secrets in CSE logs

Reply:
Yep, thanks!